environment.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
 - python~=3.7
 - aldjemy~=0.9.1
 - alembic~=1.2
+- aio-pika~=6.6.1
```

Comment:
Typically, we just update the `setup.json`. The `environment.yml` is then automatically updated by a pre-commit hook. The `requirements` files are also autogenerated, but by a Github Action workflow. The reason that this is not done by a pre-commit hook, is because it needs a complete clean and well controlled environment, so that's why it is done on the CI platform and not locally.

Reply:
Thanks! I changed every file every time before, it's really not necessary :p